index.html

```diff
@@ -6357,6 +6357,54 @@ <h4>Inappropriate Use</h4>
 specific context of their intended application.
         </p>
       </section>
+
+      <section class="informative">
+        <h3>Code Injection</h3>
+
+        <p>
+It is possible to include data in [=verifiable credentials=] that include
+executable code or scripting languages. Authors of verifiable credentials are
+advised to avoid doing so, unless necessary, and the risks have been mitigated
+to the extent possible.
+        </p>
+
+        <p>
+For example, when a single natural language string contains multiple languages
+or annotations, the contents of the string might require additional structure or
+markup in order to be presented correctly. It is possible to use markup
+languages, such as HTML, to label spans of text in different languages or to
+supply string-internal markup needed for proper display of [=bidirectional
+text=]. It is also possible to use the `rdf:HTML` datatype to encode such values
+accurately in JSON-LD.
+        </p>
+
+        <p>
+Despite the ability to encode information as HTML, implementers are strongly
+discouraged from doing this because it:
+        </p>
+
+        <ul>
+          <li>
+Requires some version of an HTML processor, which increases the burden of
+processing language and base direction information.
+          </li>
+          <li>
+Increases the security attack surface when utilizing this data model because
+naively processing HTML could result in executing a `script` tag that
+an attacker injected at some point during the data production process.
```

Comment:
```suggestion
discouraged from doing so, for the following reasons:
        </p>

        <ul>
          <li>
It requires some version of an HTML processor, which increases the burden of
processing language and base direction information.
          </li>
          <li>
It increases the security attack surface when utilizing this data model, because
naively processing HTML could result in the execution of a `script` tag that
an attacker injected at some point during the data production process.
```